xbmc/filesystem/SMBDirectory.cpp

```diff
@@ -122,6 +122,9 @@ bool CSMBDirectory::GetDirectory(const CURL& url, CFileItemList &items)
       if(StringUtils::EndsWith(strFile, "$") && aDir.type == SMBC_FILE_SHARE )
         continue;
 
+      if (StringUtils::StartsWith(strFile, "."))
+        hidden = true;
+
```

Comment:
This is a regression and should go into helix also. Interesting that it
didn't get spotted sooner :)
Den 23 feb 2015 17:59 skrev "tamland" <notifications@github.com>:

> In xbmc/filesystem/SMBDirectory.cpp
> <https://github.com/xbmc/xbmc/pull/6529#discussion_r25180337>:
>
> > @@ -122,6 +122,9 @@ bool CSMBDirectory::GetDirectory(const CURL& url, CFileItemList &items)
> >        if(StringUtils::EndsWith(strFile, "$") && aDir.type == SMBC_FILE_SHARE )
> >          continue;
> >
> > +      if (StringUtils::StartsWith(strFile, "."))
> > +        hidden = true;
> > +
>
> bool hidden = StartsWith(..? 3 lines shorter:)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/xbmc/xbmc/pull/6529/files#r25180337>.
>